bodhi/server/__init__.py

```diff
@@ -121,6 +125,38 @@ def exception_filter(response, request):
 # Bodhi initialization
 #
 
+#: An SQLAlchemy scoped session with an engine configured using the settings in Bodhi's server
+#: configuration file. Note that you *must* call :func:`initialize_db` before you can use this.
+Session = scoped_session(sessionmaker())
+
+
+def initialize_db(config):
+    """
+    Initialize the database using the given configuration.
+
+    This *must* be called before you can use the :data:`Session` object.
+
+    Args:
+        config (dict): The Bodhi server configuration dictionary.
+
+    Returns:
+        sqlalchemy.engine: The database engine created from the configuration.
+    """
+    #: The SQLAlchemy database engine. This is constructed using the value of
+    #: ``DB_URL`` in :mod:`anitya.config``.
```

Comment:
Whoops. I started out with what I just did in Anitya and missed this. Good eye! It's gone ✨